test/ufuzz.js

```diff
@@ -202,7 +191,11 @@ var ASSIGNMENTS = [
     '<<=',
     '>>=',
     '>>>=',
-    '%=' ];
+    '%=',
+];
+ASSIGNMENTS = ASSIGNMENTS.concat(ASSIGNMENTS.map(function() {
+    return '=';
```

Comment:
Would greatly prefer to just add more `'='` assignments in ASSIGNMENTS array above and dropping this code.